.github/workflows/standard_spa_pr.yml

```diff
@@ -25,45 +46,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  cache-yarn:
-    name: Cache Yarn
-    runs-on: jupiterone-dev
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Cache yarn dependencies
-        uses: actions/cache@v2
-        id: yarn-cache
-        with:
-          path: |
-            ${{ steps.yarn-cache-dir-path.outputs.dir }}
-            **/node_modules
-          key: cache-yarn-${{ hashFiles('**/yarn.lock') }}
-      - name: Install Dependencies
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
-        # Eventually we want to flip to npm and use `npm ci` here
-        run: |
-          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_AUTH_TOKEN }}" > .npmrc
-          yarn --frozen-lockfile
-
   validate:
     name: Validate
-    runs-on: jupiterone-dev
-    needs: cache-yarn
+    runs-on: ${{ inputs.runs_on }}
```

Comment:
Lets us use in-house runners but if they break fall back to GH runners (which are safe but are expensive).